src/app/containers/Article/__snapshots__/index.test.jsx.snap

```diff
@@ -295,6 +514,7 @@ exports[`ArticleContainer Component non-200 status code should render correctly
         />
       </HelmetWrapper>
       <ConsentBanner />
+      <ConsentBanner />
```

Comment:
Why is there a double `ConsentBanner`?

Reply:
![](https://media.giphy.com/media/3d87QXmWbk1vxufqhl/giphy.gif)

Reply:
![](https://media0.giphy.com/media/l2YWmdzX0DkNHk3ew/giphy.gif?cid=790b76115cae056c63412e4c7394c872)

Reply:
![](https://media.giphy.com/media/7SX1EWzetp0GVAgoqp/giphy.gif)

Reply:
Ermmmmm there is 2 `ConsentBanner`'s in each output because it's in both the top level and inside of `PageViewAnalytics`

Reply:
![](https://media3.giphy.com/media/GcSqyYa2aF8dy/giphy.gif?cid=790b76115cae07916168336c6f1d6f5f)

Reply:
`PageViewAnalytics/index.jsx` is named `ConsentBanner`, whoops